webapp/views.py

```diff
@@ -64,7 +64,15 @@ def _build_mirror_list():
         flask.request.headers.get("X-Real-IP", flask.request.remote_addr)
```

Comment:
Two lines above. `country_code` gets initialised with `NO_COUNTRY_CODE`. It's redundant and can be removed.